tools/crm_simulate.c

```diff
@@ -1028,6 +1047,7 @@ main(int argc, char **argv)
 
         cleanup_calculations(data_set);
         data_set->input = input;
+        data_set->priv = out;
```

Comment:
Yes, if we are showing neither scores nor utilizations (which happens in cts/cts-cli) then data_set->priv will be set to the new logger_out here.  Afterwards, we'll need to set it back.  Otherwise, segfaults.